web/src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import './App.css'
+import AboutUs from './components/AboutUs'
+import CarouselImages from './components/CarouselImages'
+import Faq from './components/Faq'
+import FormLab from './components/FormLab'
+import NavBar from './components/NavBar'
+import News from './components/News'
+import MapGoogle from './components/MapGoogle'
+import Contact from './components/Contact'
```

Comment:
I think we can use the relative path `@` if is possible

Reply:
Done.